uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/absence-tab/modules/settings.js

```diff
@@ -0,0 +1,11 @@
+(function (CRM) {
+  define([
+    'common/angular'
+  ], function (angular) {
+    return angular.module('absence-tab.settings', []).constant('settings', {
+      contactId: decodeURIComponent((new RegExp('[?|&]cid=([^&;]+?)(&|#|;|$)').exec(location.search)||[,""])[1].replace(/\+/g, '%20'))||null,
```

Comment:
Yes, found that this how the `contact-id`is getting fetched. Found in job roles and contact summary.